### PR TITLE
Added features

### PR DESCRIPTION
**Pull request by dhruv282**
- Added embedding feature for image grid and image slider
- Added documentation on how to use TmVis for new users
- Added instructions for timeline feature
- Added load animation on the GIF page

### CODE REVIEW

**Comment by dhruv282** [comment]
Thanks @ibnesayeed! I updated rest of the file.

File: public/scripts/embedded.js [hunk 1]
```diff
@@ -1,41 +1,72 @@
...
-    var qArray = url.split('&'); //get key-value pairs
-    for (var i = 0; i < qArray.length; i++)
-    {
-        var pArr = qArray[i].split('='); //split key and value
-        if (pArr[0] == paramName)
-            return pArr[1]; //return value
+    var imageSliderEmbed = $("#imageSlider .imageslider_wrapper").html();
+    var regex = /<h2>(.*?)<\/h2>/gm;
+    imageSliderEmbed = imageSliderEmbed.replace(regex,"")
+    imageSliderEmbed += `<link href=`+window.location.origin+`/styles/sitestyle.css" rel="stylesheet"/>
```
Comment (ibnesayeed) [inline public/scripts/embedded.js:40]: Why did you remove the opening double quote?

```suggestion
    imageSliderEmbed += `<link href="`+window.location.origin+`/styles/sitestyle.css" rel="stylesheet"/>
```

Comment (dhruv282) [reply public/scripts/embedded.js:40]: Oops! Didn't realize that. I'll add them back. Thanks!
